netflix-roulette/src/pages/home/components/Header/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import {
+  Button,
+  SearchBarContainer,
+  SearchBarText,
+  Input,
+} from "./SearchBar.styles";
+
+interface CounterProps {
+  handleSearch: (searchValue: string) => void;
+}
+
+export const SearchBar = ({ handleSearch }: CounterProps) => {
+  const [searchValue, setSearchValue] = React.useState<string>("");
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(event.target.value);
+  };
+
+  return (
+    <SearchBarContainer>
+      <SearchBarText>Find your movie</SearchBarText>
+      <div>
+        <Input
```

Comment:
You seem to be implementing [controlled input](https://reactjs.org/docs/forms.html#controlled-components), so you, probably, need to add `value = {searchValue}` to keep your app and browser in sync.

Reply:
updated, thanks